OTPKitDemo/MapView.swift

```diff
@@ -11,41 +11,47 @@ import SwiftUI
 
 public struct MapView: View {
     @StateObject private var sheetEnvironment = OriginDestinationSheetEnvironment()
-
-    @StateObject private var userLocationService = UserLocationServices.shared
-    @StateObject private var mapExtensionService = MapExtensionServices.shared
+    @ObservedObject private var locationManagerService = LocationManagerService.shared
 
     @State private var position: MapCameraPosition = .userLocation(fallback: .automatic)
 
     public var body: some View {
         ZStack {
             MapReader { proxy in
                 Map(position: $position, interactionModes: .all) {
-                    mapExtensionService
+                    locationManagerService
                         .generateMarkers()
                 }
                 .mapControls {
-                    MapUserLocationButton()
-                    MapPitchToggle()
+                    if !locationManagerService.isMapMarkingMode {
```

Comment:
I remember in one of your reviews back then, you don't prefer this negative if conditional. I'm confused about how to make this in a positive statement. Any thoughts? @aaronbrethorst

Reply:
I think this is OK. What I would worry about here is if `isMapMarkingMode` was named `notInMapMarkingMode` (or whatever) and you had to reason about that. The boolean properties should be phrased positively, it's fine to negate them.